.github/workflows/dotnetcore_build.yml

```diff
@@ -1,22 +1,25 @@
-name: Build
+name: .NET
```

Comment:
I believe that if you change the name here, it will break the nice badge we have in the README, so let's just leave this as it was before